NotifyMeCI.Engine/Managers/SettingManager.cs

```diff
@@ -60,6 +60,21 @@ public bool Minimize
             }
         }
 
+        private bool _abortedEqualsFailed = true;
+        public bool AbortedEqualsFailed
```

Comment:
For this bool property, further down there is an `Initialise` method with a switch statement about line 102 that sets the properties when the `SettingsManager` is first called. Currently this property isn't set there, so if someone were to set this to false and close the app, then re-opening it would cause the property to reset to be true again.